aria/modeling/service_instance.py

```diff
@@ -1685,12 +1725,31 @@ def inputs(cls):
     # endregion
 
     description = Column(Text)
+    relationship_edge = Column(Boolean)
     implementation = Column(Text)
+    configuration = Column(modeling_types.StrictDict(key_cls=basestring))
     dependencies = Column(modeling_types.StrictList(item_cls=basestring))
     executor = Column(Text)
     max_retries = Column(Integer)
     retry_interval = Column(Integer)
 
+    def configure(self):
+        from . import models
+        if (self.implementation is None) or (self.interface is None):
+            return
+
+        if self.plugin is None:
+            arguments = configure_operation(self)
```

Comment:
please use `execution_plugin.instantiation.configure_operation` or so, there are enough `configure_operation` calls in this module as it is and it's easily confusing :)